qmcpy/discrete_distribution/iid_std_uniform.py

```diff
@@ -19,7 +19,7 @@ class IIDStdUniform(IID):
         spawn_key       ()
     """
 
-    def __init__(self, dimension=1, seed=None):
+    def __init__(self, dimension=1, replications=1, seed=None):
```

Comment:
You don't have to do this but I'm nitpicking if I were to add a argument to a constructor I would have the new argument be always last.

Reply:
Good suggestion, just moved it